src/lsptoolshost/onAutoInsertFeature.ts

```diff
@@ -22,7 +22,7 @@ import {
 } from 'vscode-languageserver-protocol';
 
 import * as RoslynProtocol from './roslynProtocol';
-import * as UUID from 'vscode-languageclient/lib/common/utils/uuid';
+import { generateUuid } from 'vscode-languageclient/lib/common/utils/uuid';
```

Comment:
webpack didn't love that this path didn't have a `default` export. So updated the import to just pull in the used method.